pkg/authorization/apiserver/apiserver.go

```diff
@@ -120,7 +119,6 @@ func (c *completedConfig) newV1RESTStorage() (map[string]rest.Storage, error) {
 	resourceAccessReviewStorage := resourceaccessreview.NewREST(c.GenericConfig.Authorization.Authorizer, c.ExtraConfig.SubjectLocator)
 	resourceAccessReviewRegistry := resourceaccessreview.NewRegistry(resourceAccessReviewStorage)
 	localResourceAccessReviewStorage := localresourceaccessreview.NewREST(resourceAccessReviewRegistry)
-	roleBindingRestrictionStorage, err := rolebindingrestrictionetcd.NewREST(c.GenericConfig.RESTOptionsGetter)
```

Comment:
Shouldn't just the `c.GenericConfig.RESTOptionsGetter` be removed here and the `v1Storage["roleBindingRestrictions"] = roleBindingRestrictionStorage` kept intact?

Reply:
yes, you are correct, thanks